server/lib/src/be/dbentry.rs

```diff
@@ -28,6 +29,10 @@ pub struct DbEntryV2 {
 pub enum DbEntryVers {
     V1(DbEntryV1),
     V2(DbEntryV2),
+    V3 {
+        changestate: DbEntryChangeState,
+        attrs: BTreeMap<AttrString, DbValueSetV2>,
```

Comment:
Boooo, `AttrString`

Reply:
I don't know if I want to change it to Attribute in this PR here though. I have no issues with adding more migrations and a V4 in future though. That's why these are versioned, so we can always change it.

Reply:
Yeah, I know, but I'm going to boo it regardless 😛